.eslintrc.json

```diff
@@ -0,0 +1,22 @@
+{
+    "env": {
```

Comment:
My fault: I forgot to make the CI check the formatting :man_facepalming:. If I had this would have been caught, because, yes, we're using 2-space tab!

I'll fix that in a sec.

Reply:
I am not sure, but I though prettier would catch this 😞

Reply:
Not your fault. I messed up at some point.